Tools/vagrant/initvagrant.sh

```diff
@@ -1,24 +1,36 @@
 #!/bin/bash
+echo "---------- $0 start ----------"
 
 # this script is run by the root user in the virtual machine
 
 set -e
 set -x
 
+who=$(whoami)
 echo "Initial setup of SITL-vagrant instance."
+if [ $who != 'root' ]; then
+    echo "SORRY, MUST RUN THIS SCRIPT AS ROOT, GIVING UP"
+    exit 1
+fi
 
 VAGRANT_USER=ubuntu
 if [ -e /home/vagrant ]; then
     # prefer vagrant user
     VAGRANT_USER=vagrant
 fi
+echo USING VAGRANT_USER:$VAGRANT_USER
+
+cd /home/$VAGRANT_USER
+
 
 # artful rootfs is 2GB without resize:
 sudo resize2fs /dev/sda1
 
 usermod -a -G dialout $VAGRANT_USER
 
+echo "calling pre-reqs script..."
 /vagrant/Tools/scripts/install-prereqs-ubuntu.sh -y
+echo "...pre-reqs script done... initvagrant.sh continues."
```

Comment:
These are the only ones I don't like as the command will already show up in the output.